### DISCUSSION TITLE
Pesquisar Produtos (Fundamentos do Blazor com .net 8)

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#436] [Tira Dúvidas] [answered]
Pessoal, sou novato na área. 
Na lista de produtos (index.razor) que exibe os produtos e a partir dessa tela podemos excluir ou alterar, senti falta de uma pesquisa, como faria nesse caso? Imagina que tem ali uns 200 produtos como filtrar eles?
Obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @FBODeveloper, como vai?

Claro, podemos excluir ou editar sim, da mesma forma que fazemos nas páginas separadas.
Sobre a pesquisa e filtro, também é possível, inclusive com paginação (Mostro isso no curso de EF).

Na sua tentativa de implementar estes itens, onde você ficou travado? Se tiver algum código para mostrar.... consigo te ajudar melhor.

**Reply by FBODeveloper** [reply]
A pesquisa em si eu entendi (C#, EF, .net) beleza.... não entendi como colocar na página e funcionar.... eu até colocar um campo e um button para pesqusiar mas não consegui passar o conteúdo desse campo para ser pesquisado no model e posterior retorno na lista

**Reply by andrebaltieri** [reply]
Bom dia, @FBODeveloper, cola o código que não funcionou aqui que vamos ajustando! O importante é você tentar para ir evoluindo e descobrindo como conectar as coisas...
